WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -139,6 +143,21 @@ + (void)setShouldWaitForQuiescence:(BOOL)value
 
 #pragma mark Private
 
++ (NSUInteger)mjpegServerPortFromArguments
+{
+  NSArray *arguments = NSProcessInfo.processInfo.arguments;
+  NSUInteger index = [arguments indexOfObject:@"--mjpeg-server-port"];
+  if (index == NSNotFound || index == arguments.count - 1) {
+    return NSNotFound;
+  }
+  NSString *portNumberString = arguments[index + 1];
+  NSUInteger port = (NSUInteger)[portNumberString integerValue];
```

Comment:
NSString contains `unsignedIntegerValue` method

Reply:
`unsignedIntegerValue` is part of `NSNumber`, not `NSString`